admin_auto_filters/static/django-admin-autocomplete-filter/js/autocomplete_filter_qs.js

```diff
@@ -1,73 +1,73 @@
 django.jQuery(document).ready(function () {
   django.jQuery('#changelist-filter select, #grp-filters select').on(
-      'change',
-      function (e, choice) {
-          var val = django.jQuery(e.target).val() || '';
-          var class_name = this.className;
-          var param = this.name;
-          if (class_name.includes('admin-autocomplete'))
-          {
-              window.location.search = search_replace(param, val);
-          }
-      });
+    'change',
+    function (e, choice) {
+      var val = django.jQuery(e.target).val() || '';
+      var class_name = this.className;
+      var param = this.name;
+      if (class_name.includes('admin-autocomplete')) {
+        window.location.search = search_replace(param, val);
+      }
+    });
 });
```

Comment:
I’m not a developer of this project, but in general not doing whitespace changes like this is best (for easier review, git log, git blame mostly).

Reply:
Yeah, I kept it in a different commit in case that was the consensus. I'm fine either way - whatever folks prefer.